app/commands.py

```diff
@@ -611,10 +616,31 @@ async def requests(ctx: Context) -> Optional[str]:
             l.append(f"Failed to find requested map ({map_id})?")
             continue
 
-        l.append(f"[{p.embed} @ {dt:%b %d %I:%M%p}] {bmap.embed}.")
+        l.append(f"#{id}: {bmap.embed}\n({dt:%b %d %I:%M%p}) Requested for "
+                 + f"{requested_status} by {p.embed}. "
+                 + f"{'No additional comment was specified.' if comment == '' else f'Additional comment: {comment}'}")
 
     return "\n".join(l)
 
+@command(Privileges.NOMINATOR, aliases=["dr"], hidden=True)
```

Comment:
seems like nice middleground
```suggestion
@command(Privileges.NOMINATOR, aliases=["decline", "dr"], hidden=True)
```